Game/MyProject/Game/ModelLoader/ModelLoader.hpp

```diff
@@ -0,0 +1,29 @@
+//
+//  ModelLoader.hpp
+//  Game
+//
+//  Created by Tamas Both - (p) on 09/04/2019.
+//
+
+#pragma once
+
+#include <iostream>
+
+#include <assimp/Importer.hpp>
+#include <assimp/scene.h>
+#include <assimp/postprocess.h>
+
+#include "SceneNode.hpp"
+#include "../ResourceManager/ResourceManager.hpp"
+
+class ModelLoader {
+public:
+    // public function(s) and method(s)
+    static std::shared_ptr<SceneNode> loadModel(const std::string&, const std::string&);
+    
+private:
+    // private function(s) and method(s)
+    static void processNode(aiNode*, const aiScene*, SceneNode*, const std::string&);
```

Comment:
Do a ModelLoaderUtils namespace and add them there instead of static functions, they play better with multiple translation unity.